shiptendance/views/show.ejs

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title></title>
+</head>
+<body>
+  <h1><%= pirate.name %></h1>
+  <p><%= pirate.catchphrase %></p>
+
+  <form action="/recipes/<%= pirate._id %>" method="post">
```

Comment:
Looks like this was copied from the cookbook example - you would need to make sure to change recipes to pirates. Also, you probably want to keep this commented out until the route for the delete functionality has been set up in your pirates.js file.